appveyor.yml

```
@@ -83,8 +76,8 @@ build_script:
   - pip install wheel numpy
```

```suggestion
  - pip install wheel numpy cython
```

(that's the reason appveyor once are failing right now)